README.md

```diff
@@ -24,7 +24,16 @@ It provides information such as:
 
 ## Installing/ updating
 
-A PDF containing installation instructions is attached to each [release](/../../releases).
+These applications can be installed via an interactive Qlik Application Automation [installer](https://community.qlik.com/t5/Official-Support-Articles/Qlik-Cloud-Monitoring-Apps-Workflow-Guide/ta-p/2134140) or [manually](/../../releases) using the PDF guide.
+
+## Tenant Subscription Support
+
+Certain monitoring apps only support specific subscription types. Refer to the subscription support matrix for this app below.
```

Comment:
```suggestion
## Subscription Compatibility

Some monitoring apps are designed for specific Qlik Cloud subscription types. Refer to the subscription compatibility matrix for this app below.
```
Compatibility ++ design

Reply:
Can we instead link to the parent repo [here](https://github.com/qlik-oss/qlik-cloud-monitoring-apps?tab=readme-ov-file#applications)?